opennlp-tools/pom.xml

```diff
@@ -81,7 +81,6 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <argLine>@{argLine}</argLine>
```

Comment:
Rebase ur code, this is already removed in present trunk and shuldn't be in this PR